src/Microsoft.VisualStudio.ProjectSystem.IntegrationTests/Utilities/NuGetApexTestService.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using System.Linq;
+using System.Threading;
+using EnvDTE;
+using Microsoft.Test.Apex;
+using Microsoft.Test.Apex.VisualStudio;
+using Microsoft.VisualStudio.Shell.Interop;
+using NuGet.SolutionRestoreManager;
+using NuGet.VisualStudio;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS
+{
+    [Export(typeof(NuGetApexTestService))]
+    public class NuGetApexTestService : VisualStudioTestService<NuGetApexVerifier>
```

Comment:
Did you write these files or did they come from the NuGet team?  Are we going to maintain them indefinitely?

Reply:
We should talk to the nuget team about getting these merged back into APEX. They currently exist [here](https://github.com/NuGet/NuGet.Client/tree/dev/test/NuGet.Tests.Apex/NuGet.Tests.Apex/Fixtures)